libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2127,6 +2130,72 @@ void GCS_MAVLINK::send_raw_imu()
 #endif
 }
 
+#if AP_MAVLINK_MSG_HIGHRES_IMU_ENABLED
+void GCS_MAVLINK::send_highres_imu()
+{
+    uint16_t fields_updated = HIGHRES_IMU_UPDATED_NONE;
+    Vector3f accel = Vector3f();
+    Vector3f gyro = Vector3f();
+    uint8_t accel_id = 0;
+#if AP_INERTIALSENSOR_ENABLED
```

Comment:
i'd prefer not to send the msg at all if AP_INERTIALSENSOR_ENABLED is false. That would allow us to simplify the code quite a bit, so less flash